### ISSUE TITLE
AGH v0.107.29 on Docker doesn't work

### ISSUE THREAD

**Issue by boardlord1** [AdguardTeam/AdGuardHome#5751]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, Other (please mention the version in the description)

### CPU architecture

AMD64

### Installation

Docker

### Setup

On one machine

### AdGuard Home version

0.107.29

#### What did you do?
I've updated AGH via Portainer as usual (recreate image by re-pulling). I've been using AGH On Synology NAS for many months without problems. 

#### Expected result
Like mane times before, a simple update.

#### Actual result
The logs don't display any errors, AGH starts, binds addresses successfully.

However, it doesn't resolve addresses, and the AGH GUI doesn't load on 192.168.7.124:8080 - just times out.

#### Additional information
My Docker compose in my stack for AGH 
```
  adguard:
    image: adguard/adguardhome
    container_name: "adguardhome"
    cap_add:
      - NET_ADMIN
    networks:
      composemacvlan:
        ipv4_address: 192.168.7.124
    environment:
      - PGID=100
      - PUID=1037
      - TZ=Europe/Budapest
    privileged: true
    volumes:
      - "/volume1/docker/adguard/conf:/opt/adguardhome/conf:rw"
      - "/volume1/docker/adguard/work:/opt/adguardhome/work:rw"
```

**Comment by ainar-g** [comment]
Portainer is most likely affected by the same thing as #5711 and #5713.  It seems like some Docker-related software doesn't automatically update entrypoints, and so the container must be removed and added again for such software to register the change.

**Comment by boardlord1** [comment]
@ainar-g I don't have a problem with AGH v0.107.28 - that's what I've downgraded to to make it work again. Only happened with v0.107.29. And as I said it didn't say in the logs "[FATAL tini (7)] exec -h failed: No such file or directory". JaZo's suggestion to change the cmd to isn't relevant as that's what Portainer tells me it's correct:
![image](https://user-images.githubusercontent.com/8179205/233054864-dd0b4d07-3a4a-4f84-a709-d04189c2633c.png)

**Comment by ainar-g** [comment]
I see.  If you enable [verbose logs][1], do you see any additional messages like warnings?  Is AdGuard Home available from inside the container?

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Comment by boardlord1** [comment]
Wow... Talk about ghost in the machine!

In order to gather data for you, I stopped v0.107.28, deleted container and image, and re-deployed my stack with v0.107.29. And now it works... So this must have been a fluke, I'll close this as solved now.

**Comment by boardlord1** [comment]
@ainar-g Found the exact issue, it was portainer's fault. When I updated my Plex container a few days ago (recreate the container with the "pull the latest image" toggle enabled), the new container listened on a different IP address to the one specified in the docker compose file (192.168.7.112 vs .123). I'm pretty sure that's what happened with my AGH when I opened this issue.

So in all, portainer bug for sure.
